recipes/alps/bld.bat

```diff
@@ -0,0 +1,84 @@
+
+:: Compile clapack
+"%PYTHON%" "%RECIPE_DIR%\get_clapack_src.py"
+if errorlevel 1 exit 1
+
+mkdir build_clapack
+if errorlevel 1 exit 1
+cd build_clapack
+if errorlevel 1 exit 1
+
+cmake ^
+    -G "%CMAKE_GENERATOR%" ^
+    -DCMAKE_INSTALL_PREFIX="%SRC_DIR%\ext" ^
+    -DBoost_ROOT_DIR="%LIBRARY_INC%" ^
+    ../clapack
+if errorlevel 1 exit 1
+
+cmake --build . --config Release -- /verbosity:minimal
+if errorlevel 1 exit 1
+
+cmake --build . --config Release --target install
+if errorlevel 1 exit 1
+
+
+
+:: Start actual ALPS build
+cd %SRC_DIR%
+if errorlevel 1 exit 1
+
+
+mkdir build
+if errorlevel 1 exit 1
+
+cd build
+if errorlevel 1 exit 1
+
+set PYTHON_LIBRARY=%PREFIX%\libs\python%PY_VER:~0,1%%PY_VER:~2,1%.lib
+
+
+cmake ^
```

Comment:
This package is not building in less than the max 90 min we have in AppVeyor. I am not sure it will be possible to merge this.